src/entitytypes.js

```diff
@@ -994,6 +1023,34 @@ SlugIdArray.prototype.remove = function(slug) {
   return false;
 };
 
+/**
+ * The slice() method extracts a portion of a SlugIdArray's buffer into a
+ * new buffer from begin to end (end not included).
+ * Specifying negative indexes causes the slice to be generated relative to the end.
```

Comment:
Isn't `slice()` to return a copy of a slice of the array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Note I suggest we implement it the same way JS does... for consistency